### PR TITLE
Basic Endpoints

### PR DESCRIPTION
**Pull request by TomRafferty**
## Description

I've added some primitive endpoints to our back-end api to help with future tasks.
These include things like retrieve users and retrieve users with dietary ristrictions

### CODE REVIEW

**Review by Johng117 — APPROVED** [review]
LGTM.

File: server/api.js [hunk 1]
```diff
@@ -1,9 +1,87 @@
-import { Router } from "express";
+import { response, Router } from "express";
+import query from "./db";
 
 const router = Router();
+const client = query; //just looks a little nicer than query.query
```
Comment (CanaryWharf) [inline server/api.js:5]: You can do `import query as client from './db'` instead. Looks a bit cleaner and makes sure imports are at the top

File: server/api.js [hunk 2]
```diff
@@ -1,9 +1,87 @@
-import { Router } from "express";
+import { response, Router } from "express";
```
Comment (milleymilli) [inline server/api.js:1]: What is this response? I didn't get and where are you using it.

**Review by milleymilli — CHANGES_REQUESTED** [review]
Try to replay comments. And good job.

File: server/api.js (hunk 2 above)
Comment (TomRafferty) [reply server/api.js:1]: Whoops must have accidentally imported it. Usually happens when I miss type something and it autocorrects, such as the kebab incident 😂

File: server/api.js [hunk 3]
```diff
@@ -1,9 +1,50 @@
...
+		})
+		.catch((error) => {
+			console.error(error);
+			res.status(error.status).send(error);
+		});
+});
+
+//retrieve user by id
+router.get("/users/:userId", (req, res) => {
+	const id = req.params.userId;
```
Comment (milleymilli) [inline server/api.js:48]: What if the client starts querying using  NAN **userId** ??

**Review by milleymilli — CHANGES_REQUESTED** [review]

**Pull request closed without merging**